mono.yml

```diff
@@ -0,0 +1,15 @@
+---
+language: custom
+repo: "https://github.com/appsignal/appsignal-python"
+bootstrap:
+  command: ""
+clean:
+  command: "hatch clean"
+build:
+  command: "hatch build"
+publish:
+  command: "hatch publish"
+test:
+  command: "hatch run test:pytest"
+read_version: "script/read_package_version"
+write_version: "script/write_package_version"
```

Comment:
Unless I'm missing something, I believe this is re-implementing the `hatch version` command: https://hatch.pypa.io/latest/version/

Reply:
> (Instead of read_version and write_version, you may want to just have a single version custom script? Hatch's behaviour of printing it if no argument is passed and updating it if an argument is passed seems reasonable-ish to me)

I figured I make two config options so you can write specific scripts and don't have to check if the script received an argument or not.